chingu-voyages/v50-tier2-team-16
# Feature/initialize react and tailwind

## What type of PR is this? (check all applicable)

## Description
Initializes React app using Vite, and also initializes Tailwind CSS. I intentionally deleted css in App.css to avoid potentially conflicting styles in the future. 
In addition, my teammate info was added to the README file.

## QA Instructions, Screenshots, Recordings
run command: "npm run dev" in terminal, it should look like this:

![image](https://github.com/chingu-voyages/v50-tier2-team-16/assets/138247163/f96803c3-0692-4d01-b7cc-7cf6175ab155)


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![til](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExbGhjaWFjaGowdGNvNjJ6aXR2ZnVjcHpwcnpwanZ3cWhhYmx1YmJoYiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/5zf2M4HgjjWszLd4a5/giphy.gif)
